docs/pages/includes/role-spec.mdx

```diff
@@ -36,6 +36,8 @@ spec:
     # permit_x11_forwarding allows users to use X11 forwarding with openssh
     # clients and servers through the proxy
     permit_x11_forwarding: true
+    # device_trust_mode enforces authenticated device access for the owner of this role.
+    device_trust_mode: required|optional|off
```

Comment:
Do we have a convention in place of showing all options separated by `|`? Is there a default value when this isn't specified in a role?

Reply:
Default value is optional.

Regarding convention, I just followed the pattern from the same page:
```
   record_session:
      desktop: true
      default: best_effort|strict
      ssh: best_effort|strict
```

but on the same page, we also have options separated with comma and inside square brackets.
```
    join_sessions:
      - name: Auditor oversight
        roles : ['prod-access']
        kinds: ['k8s', 'ssh']
        modes: ['moderator', 'observer']
```

Reply:
`|` is generally used to mean "or" on this page.

The options _separated with comma inside square brackets_ is just literal YAML list notation.

Reply:
The default is "off" for OSS and "optional" for Enterprise. Also, OSS can never use a mode that isn't "off".